plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointIcon.tsx

```diff
@@ -11,6 +12,7 @@ const PointIcon: React.FC<BaseFieldProps<"pointIcon">> = ({
 }) => {
   const [imageURL, setImageURL] = useState(value.url ?? "");
   const [imageSize, setImageSize] = useState(value.size);
+  const [imageSizeInMeters, setImageSizeInMeters] = useState(value.imageSizeInMeters ?? true);
```

Comment:
Unless @lavalse or @rot1024 said it should default to true, I feel like this should default to false, so `!!value.imageSizeInMeters` works here

Reply:
Also, imageSizeInMeters is the API's field. For our purposes here in saving the value, simplifying is preferable. So could you make it value.sizeInMeters

Reply:
done